trade-finance-manager-api/api-tests/v1/utilisation-report-service/update-utilisation-report-service.api-test.ts

```diff
@@ -0,0 +1,152 @@
+import api from '../../../src/v1/api';
+import app from '../../../src/createApp';
+import createApi from '../../api';
+import testUserCache from '../../api-test-users';
+import { ReportWithStatus } from '../../../src/types/utilisation-report-service';
+import { UTILISATION_REPORT_RECONCILIATION_STATUS } from '../../../src/constants';
+
+jest.unmock('../../../src/v1/api');
```

Comment:
I am not a fan of needing to do this, but without this line the function `api.updateUtilisationReportStatus` does not exist